test/helpers/wrappers.go

```diff
@@ -104,6 +104,24 @@ func CurlWithHTTPCode(endpoint string, optionalValues ...interface{}) string {
 		CurlConnectTimeout, endpoint)
 }
 
+// CurlWithRetries returns the string representation of the curl command that
+// retry the request if transient problems occur. Parameter retries are indicated the maximum retry times.
```

Comment:
Nit:

```suggestion
// CurlWithRetries returns the string representation of the curl command that
// retries the request if transient problems occur. The parameter "retries"  indicates the maximum number of attempts.
```

Also, could you reformat this comment to be consistent with the rest of the file? This comment block is wider than the rest (`gq` if you're using vim ;) )